testing/src/runner/runner.py

```diff
@@ -93,7 +95,7 @@ def run_tests(self) -> None:
         print(f"Running {len(self.config.tests)} tests ...\n")
         print("--------------------------------\n")
 
-        failed_tests = []
+        failed_tests: List[str] = []
```

Comment:
Now, (since 3.9 I think) you can use `list[str]` without the need to import it, also for `dict`